roles/kuryr/templates/controller-deployment.yaml.j2

```diff
@@ -9,6 +9,11 @@ metadata:
   namespace: {{ kuryr_namespace }}
 spec:
   replicas: 1
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
```

Comment:
should this be rollingParams? https://docs.openshift.com/container-platform/3.11/dev_guide/deployments/deployment_strategies.html#rolling-strategy

Reply:
ohh, I now see this is a Deployment object instead of a DeploymentConfig

Reply:
Yup!